tools/runqslower.py

```diff
@@ -240,7 +260,7 @@ def print_event(cpu, data, size):
     b.attach_kprobe(event="finish_task_switch", fn_name="trace_run")
 
 print("Tracing run queue latency higher than %d us" % min_us)
-print("%-8s %-16s %-6s %14s" % ("TIME", "COMM", "PID", "LAT(us)"))
+print("%-8s %-16s %-6s %14s" % ("TIME", "COMM", "TID", "LAT(us)"))
```

Comment:
This does not sound right. You cannot use "TID" all the time. If user specify '-p <pid>', you want to show "PID".

Reply:
We might get threads showing up in case of '--pid' flag too. So using PID is also not correct in that case either.
Anyways my 2 cents are on just to simply stay with showing PID as a column value even for threads. Htop, top does the same. Showing PID/TID is bringing quite a lot possible confusion here.
Or what do you vote for?
If you prefer I can make it show PID if '--pid' and TID if  '--tid'.

Reply:
Think about this again. For kernel run queue, actually TID indeed makes sense. So, yes, we can just do TID here.  If user filtered through pid, showing TID should be okay too, redundant PID information may not be needed. If user did not do any filterring, TID is the right choice too.

In summary, we can just use "TID" as you changed the above. Please a comment though. Please address other comments and resubmit. Thanks!

Reply:
Thanks, just to make it clear :)
- In case `--pid` cmd line arg -> show `PID` in the report column
- In case `--tid` cmd line arg specified -> show `TID` in the report column
Is the above what you think would make the most sense?